openedx/adg/lms/applications/api/serializers.py

```diff
@@ -103,3 +103,12 @@ class Meta:
         read_only_fields = (
             'id', 'created', 'modified',
         )
+
+
+class ReferenceSerializer(serializers.ModelSerializer):
+    """
+    Serializer for the `Reference` model
+    """
+    class Meta:
+        model = Reference
+        fields = ['id', 'name', 'position', 'relationship', 'phone_number', 'email', 'user_application']
```

Comment:
Taken from DRF [documentation](https://www.django-rest-framework.org/api-guide/serializers/):

"It is strongly recommended that you explicitly set all fields that should be serialized using the fields attribute. This will make it less likely to result in unintentionally exposing data when your models change."